### DISCUSSION TITLE
Is this package actually RN-specific?

### DISCUSSION THREAD

**Discussion by JulianKingman** [goatandsheep/react-native-dotenv#158] [Q&A] [answered]
I'm looking to include this package in an internal package I'm using, which is a utility library used by both react native and web. I want to make sure it works well in both environments, reading through the source code, it doesn't look like this should be a problem.

The name of the package makes it a little confusing, as I would assume it's RN only.

Thanks!
Julian

**Comment by goatandsheep** [comment, marked as answer]
It's not. It's a babel plugin! I'm working on a course that explains why it should work on any babel based environment that needs this.
